readthedocs/notifications/messages.py

```diff
@@ -232,8 +232,9 @@ def get_rendered_body(self):
         header=_("No HTML content found"),
         body=_(
             textwrap.dedent(
-                f"""
-             No "{BUILD_COMMANDS_OUTPUT_PATH_HTML}" folder was created during this build.
+                """
+             No "$READTHEDOCS_OUTPUT/html" folder was created during this build.
+             Read more about <a href="https://docs.readthedocs.io/page/reference/environment-variables.html#envvar-READTHEDOCS_OUTPUT">this environment variable</a>.
```

Comment:
Thinking about why I didn't like this error message more, I don't think mentioning the environment variable makes it more clear.

The underlying issues with the original message:

- The problem we are trying to solve here is not that the path doesn't exist, but that the path *was empty*. We should be clear about this in the error message
- The fact that the path doesn't exist is a little bit of a problem too. Though, I feel like creating the paths is something we could do and save users an obvious step.

That is, close to:

```suggestion
             No content was output to the path `_readthedocs/html`.
             You can use the <a href="https://docs.readthedocs.io/page/reference/environment-variables.html#envvar-READTHEDOCS_OUTPUT">READTHEDOCS_OUTPUT environment variable</a> to output files to an absolute path.
```

Reply:
We don't want to expose this `_readthedocs/html` since we don't want users to use that folder because it could change in the future. Using the `$READTHEDOCS_OUTPUT` is the correct way.

I think we could use something like this instead:

```suggestion
             No content was output to the path "$READTHEDOCS_OUTPUT/html".
             Read more about <a href="https://docs.readthedocs.io/en/stable/build-customization.html#where-to-put-files">where to put your built files</a>.
```

Reply:
Yeah that seems like an improvement. I like linking to the build customization documentation, it's a better resource for this error.

I'd update that link to use the `/page/` URL pattern, but otherwise this looks good :+1:

Reply:
```suggestion
             No content was output to the path "$READTHEDOCS_OUTPUT/html".
             Read more about <a href="https://docs.readthedocs.io/page/build-customization.html#where-to-put-files">where to put your built files</a>.
```